symfony/framework-bundle/3.3/config/services.yaml

```diff
@@ -1,6 +1,9 @@
 # Put parameters here that don't need to change on each machine where the app is deployed
 # https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
 parameters:
+    router.request_context.host: '%env(ROUTER_REQUEST_CONTEXT_HOST)%'
+    router.request_context.scheme: '%env(ROUTER_REQUEST_CONTEXT_SCHEME)%'
+    router.request_context.base_url: '%env(ROUTER_REQUEST_CONTEXT_BASE_URL)%'
```

Comment:
if we want these, shouldn't we move them in `config/packages/framework.yaml`?
I'm not sure the env var actually need to be defined, what about setting their default value to `null` also?

Reply:
I agree. Reducing the number of env vars is a must (we're receiving more and more messages from people annoyed by them because maybe we unintentionally overused them).